### PR TITLE
170 dark mode

### PR DESCRIPTION
**Pull request by ademDurakovic**
#170 

The more page was pretty much pitch black when the dark mode switch was on essentially making the dark mode switch useless. These changes allow for a working dark mode.

Technically, I used the MorePage.tsx and updated the JSX to match the theme (so it is dynamic). 

This PR also contains a test to see if the "Logout" button has been rendered.

### CODE REVIEW

File: __tests__/MoreScreen.test.tsx [hunk 1]
```diff
@@ -1,25 +1,37 @@
...
+  },
+  theme: {
+    darkMode: false, // Add mock theme state if needed
+  },
+});
+
+// Mock the ThemeProvider with spies
 jest.mock('../lib/components/ThemeProvider', () => ({
-  useTheme: () => ({
+  useTheme: jest.fn(() => ({
```
Comment (AmarHadzic) [inline __tests__/MoreScreen.test.tsx:24]: Using jest.fn() to mock the function. Can make this a "variable" so it can be used across multiple tests

**Review by AmarHadzic — COMMENTED** [review]
Looks really good overall. I just added some suggestions, but everything looks like its good and it works

**Comment by kate-holdener** [comment]
> #170
> 
> The more page was pretty much pitch black when the dark mode switch was on essentially making the dark mode switch useless. These changes allow for a working dark mode.
> 
> Technically, I used the MorePage.tsx and updated the JSX to match the theme (so it is dynamic).
> 
> This PR also contains a test to see if the "Logout" button has been rendered.

A screenshot of what the updated version looks like would be helpful.

**Comment by rcAsironman** [comment]
@ademDurakovic Good job! You are on right track. Can you also make changes to the Logout button color? when in dark mode, apply a white background to the button and set the text color to black, and vice versa. Also, please try to attach the output of the work you have done.

Reference
IMG_0060

**Comment by ademDurakovic** [comment]
Screenshot 2024-10-06 at 2 27 55 PM
Screenshot 2024-10-06 at 2 27 55 PM

This is how the button looks after the most recent push.

Thanks

**Review by rcAsironman — APPROVED** [review]
lgtm;

**Review by yashb196 — APPROVED** [review]

**Pull request merged**